spec/requests/api/v1/posters_request_spec.rb

```diff
@@ -190,5 +190,73 @@
     posters = JSON.parse(response.body, symbolize_names: true)
     expect(posters[:meta][:count]).to eq(posters[:data].count)
   end
+
+  it "sorts results ascending by query parameters" do
```

Comment:
it block will check the order of posters in response body by comparing response's first & last posters's ids data set order to first & last poster's ids. the order should be the same for ascending ordering.